lib/models/IonPublicKeyModel.ts

```diff
@@ -1,11 +1,13 @@
 import IonPublicKeyPurpose from '../enums/IonPublicKeyPurpose';
+import JwkEd25519 from '../models/JwkEd25519';
+import JwkEs256k from '../models/JwkEs256k';
 
 /**
  * Data model representing a public key in the DID Document.
  */
 export default interface IonPublicKeyModel {
   id: string;
   type: string;
-  publicKeyJwk: object;
+  publicKeyJwk: JwkEs256k | JwkEd25519;
```

Comment:
Not sure the strong type enforcement is right here. It is intentionally `object` as we do not want to dictate what users put in here as their public keys, they can chose whatever they want.

Reply:
Within reason right? We still want to adhere to the JWK data model IIRC

Reply:
@kdenhartog, yes the intention is to allow any JWK, but since JWK is such an open-ended spec, with only `kty` being the required property, I ended up not creating yet another model for it in Sidetree core reference implementation, thus as well as the ION SDK. A case can be made to enforce `kty`, and you can file a separate if you think that's the case, but the point here is that it should not be restricted to just the above two key types.

Reply:
Yeah makes sense - My take was similar to yours that we don't want to overly restrict it as well

Reply:
Reverted back to `object` type: https://github.com/decentralized-identity/ion-sdk/pull/20/commits/d63d081902df1691bc1f9b1c27f4d204e43d7519